website/docs/guides/custom-selections.md

```diff
@@ -1,17 +1,14 @@
 # Custom Selections
 
-In the [Selecting Days tutorial](/basics/selecting-days). we have set different
-modes to change the selection behavior. Under the hood, DayPicker handles the
-interaction with the days, assigning them the `selected` modifier.
+In the [Selecting Days guide](/basics/selecting-days). we have set different
+modes to change the selection behavior.
 
-To implement a selection behavior by yourself, you can either switch
-to custom selection mode or – to keep the built-in modes – use a custom Day component.
+To implement a selection behavior by yourself, you can either switch to custom
+selection mode or – to keep the built-in modes – use a custom Day component.
 
 ## Switching to the custom mode
 
-1. set the selection mode to `mode="custom"`
-2. store the selected dates in the parent’s state
-3. use the `selected` and `onDayClick` props
+To switch to custom selection, the `mode` propo to `custom`:
```

Comment:
“set the” … “prop”…

Reply:
Thanks fixed in b508d948432d138abed260923935b652f7a5fca4.